checker/tests/initialization/TestPolyInitialized.java

```diff
@@ -0,0 +1,36 @@
+import org.checkerframework.checker.initialization.qual.Initialized;
+import org.checkerframework.checker.initialization.qual.NotOnlyInitialized;
+import org.checkerframework.checker.initialization.qual.PolyInitialized;
+import org.checkerframework.checker.initialization.qual.UnknownInitialization;
+
+public class TestPolyInitialized {
+
+    @NotOnlyInitialized String testStr;
+
+    String test = "test";
+
+    TestPolyInitialized(@UnknownInitialization String str) {
+        this.testStr = str;
+    }
+
+    @PolyInitialized String identity(@PolyInitialized String str) {
+        return str;
+    }
+
+    void test1() {
+        identity(testStr);
```

Comment:
You need to assign/return the result of the call to something of a known type, to make sure you actually get back what you want.

Reply:
done!